packages/replay/src/replay.ts

```diff
@@ -1100,6 +1101,23 @@ export class ReplayContainer implements ReplayContainerInterface {
       return;
     }
 
+    const start = this._context.initialTimestamp;
+    const now = Date.now();
+    const duration = now - start;
+
+    // If session is too short, or too long (allow some wiggle room over maxSessionLife), do not send it
+    // This _should_ not happen, but it may happen if flush is triggered due to a page activity change or similar
+    if (duration < this._options.minReplayDuration || duration > this.timeouts.maxSessionLife + 5_000) {
```

Comment:
Should we have different logic if replay is too long? e.g. it needs to start a new session (though that logic could be where the bug is).

Reply:
I mean, we have a bunch of logic for this (and tests...), so ideally it should really not happen (but it is 😬 ). IMHO I'd get out this fix, I also added some logic to log out when this happens, so maybe we can look at this then to help figure out _why_ this even happens.

Reply:
I'll add another log for this! 👍